resolwe_bio/processes/workflows/wgs.yml

```diff
@@ -0,0 +1,474 @@
+- slug: workflow-wgs
```

Comment:
Modify the slug to indicate that it handles paired-end data - `workflow-wgs-paired`